tools/NuGet/template-artifactory/DynamoVisualProgramming.DynamoCoreSymbols.nuspec

```diff
@@ -13,7 +13,7 @@
     <copyright>Copyright Autodesk 2019</copyright>
   </metadata>
   <files>
-    <file src="*.pdb" target="lib\net47" />
+    <file src="*.pdb" target="lib\net47" exclude="CefSharp*.pdb;Helix*.pdb;MIConvexHull.NET Standard.pdb"/>
```

Comment:
hmm, can we pull this out into a variable at the top of the file? we're going to forget to update this list.

Reply:
I don't think we can do this because this is an XML with schema defined here: https://docs.microsoft.com/en-us/nuget/reference/nuspec#general-form-and-schema